test/cpp/jit/test_gpu.cpp

```diff
@@ -10627,6 +10627,189 @@ TEST(NVFuserTest, FusionGetComputeAtRelPos_CUDA) {
   }
 }
 
+TEST(NVFuserTest, FusionWelfordOp_CUDA) {
+  Fusion fusion;
+  FusionGuard fg(&fusion);
+
+  int M = 64, N = 128;
+
+  auto tv0 = makeSymbolicTensor(2);
+  fusion.addInput(tv0);
+  auto tv1 = mul(tv0, new Double(1));
+  std::vector<TensorView*> tvs = Welford(tv1, {1});
```

Comment:
I'm wondering if it would make more sense to return a struct with fields named like `avg` and `var`, instead of a plain vector.

Reply:
I agree especially on the API part. Added `WelfordResult` as the return type. Thanks.